packages/create-sitecore-jss/src/templates/vue/scripts/generate-config.js

```diff
@@ -13,13 +13,13 @@ const packageConfig = require('../package.json');
  * NOTE! Any configs returned here will be written into the client-side JS bundle. DO NOT PUT SECRETS HERE.
  * @param {object} configOverrides Keys in this object will override any equivalent global config keys.
  */
-// VUE_APP_JSS_APP_NAME env variable has been deprecated since v.21.6, VUE_APP_SITE_NAME should be used instead
+// VUE_APP_JSS_APP_NAME env variable has been deprecated since v.21.6, VUE_APP_SITECORE_SITE_NAME should be used instead
 module.exports = function generateConfig(configOverrides) {
   const defaultConfig = {
     sitecoreApiKey: 'no-api-key-set',
     sitecoreApiHost: '',
     jssAppName: process.env.VUE_APP_JSS_APP_NAME,
```

Comment:
Same comment as for Angular (above), do we want to remove this variable, and instead set value only in _sitecoreSiteName_?

Reply:
Agree, thought that was confusing as well. I removed them.